tests/test_pragma_parser.py

```diff
@@ -8,93 +8,105 @@
 )
 
 
-def test_simple_pragma():
+def test_simple_pragma() -> None:
     comment = "#pylint: disable = missing-docstring"
     match = OPTION_PO.search(comment)
-    for pragma_repr in parse_pragma(match.group(2)):
-        assert pragma_repr.action == "disable"
-        assert pragma_repr.messages == ["missing-docstring"]
+    if match:
+        for pragma_repr in parse_pragma(match.group(2)):
+            assert pragma_repr.action == "disable"
+            assert pragma_repr.messages == ["missing-docstring"]
```

Comment:
```suggestion
    assert match
    for pragma_repr in parse_pragma(match.group(2)):
        assert pragma_repr.action == "disable"
        assert pragma_repr.messages == ["missing-docstring"]
```
Or something similar because we want the test to actually fail if there is no match.

Reply:
Give me a minute!